go/vt/sqlparser/comments.go

```diff
@@ -238,7 +240,7 @@ func ExtractCommentDirectives(comments Comments) CommentDirectives {
 }
 
 // IsSet checks the directive map for the named directive and returns
-// true iff the directive is set and has a true/false or 0/1 value
+// true if the directive is set and has a true/false or 0/1 value
```

Comment:
This wasn't a typo :) iff == if and only if

Reply:
TIL. I changed to if and only if to be super clear. However, if you think `iff` is a pretty standard abbreviation, I can change it back.